src/widgets/_widgets.scss

```diff
@@ -53,6 +53,11 @@ dialog {
             @include shadow(2);
         }
     }
+
+    > headerbar.titlebar label.title {
+        font-size: 0.1px;
```

Comment:
Does this still not work correctly in more recent versions of GTK if we just set 0px?

Reply:
I'm not sure, but I can't remember exactly what the issue was before either. I can add a comment